src/prefect/tasks/mysql/mysql.py

```diff
@@ -0,0 +1,192 @@
+import pymysql.cursors
+
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class MySQLExecute(Task):
+    """
+    Task for executing a query against a MySQL database. 
+    Args:
+        - db_name (str): name of MySQL database
+        - user (str): user name used to authenticate
+        - password (str): password used to authenticate
+        - host (str): database host address
+        - port (int, optional): port used to connect to MySQL database, defaults to 5432 if not provided
+        - query (str, optional): query to execute against database
+        - data (tuple, optional): values to use in query, must be specified using placeholder in query string
+        - commit (bool, optional): set to True to commit transaction, defaults to false
+        - charset (str, optional): charset you want to use (defaults to utf8mb4)
+        - **kwargs (dict, optional): additional keyword arguments to pass to the
+            Task constructor
+    """
+
+    def __init__(
+        self,
+        db_name: str,
+        user: str,
+        password: str,
+        host: str,
+        port: int = 5432,
```

Comment:
FYI the default in these spots should be 3307, as that is the default mysql port